synapse/storage/databases/main/stream.py

```diff
@@ -742,7 +753,7 @@ def _f(txn):
                 " LIMIT 1"
             )
             txn.execute(sql, (room_id, stream_ordering))
-            return txn.fetchone()
+            return cast(Tuple[int, int, str], txn.fetchone())
```

Comment:
Is this correct? Couldn't the query select 0 rows, so that `txn.fetchone() is None`?

Reply:
Oh. This was a mistake.